app/validators/claim_textfield_validator.rb

```diff
@@ -67,7 +67,7 @@ def validate_advocate_category
 
 # must be present unless case type is breach of crown court order
```

Comment:
:+1: nope its not. can i remove this in a subsequent branch - just to avoid the flickering tests which took about 2 hours to overcome this time round